packages/nocodb/src/db/relation-update-webhook-handler.ts

```
@@ -0,0 +1,84 @@
+  async sendBeforeUpdateWebhook() {
+    this.parentUpdateWebhookHandler = await UpdateWebhookHandler.beginUpdate(
+      {
+        context: this.relationWebhookContext.context,
+        user: this.relationWebhookContext.user,
+        baseModel: this.relationWebhookContext.parentBaseModel,
+        isSingleUpdate: true,
+        ignoreWebhook: this.relationWebhookContext.ignoreWebhook,
+      },
+      this.parentRowId,
+    );
+    if (this.childRowId != this.childRowId) {
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)